File: Assignment7.java
/*
 *Matthew Frankowski
 * CUS1168
 *Assignment 7
 * */
public class Assignment7 {

    public static void main(String[] args) {

        double j = 1, k = (j + 13.0) / 27.0, i; /*Initialize variables: j, k*/

        while (k <= 10.0){ /*Loop while k is less than or equal to 10.0*/

            k = k + 1.2; /*Increment k by 1.2 in each iteration of the loop*/
            i = 3.0 * k - 1.0; /*Calculate the value of j based on the current value of k*/

            System.out.println("k: " + k + ", i: " + i); /*Print the values of k and i in each iteration of the loop*/
        }

    }

}

File: Assignment 7.rb
# Matthew Frankowski
# CUS 1168
# Assignment 7

# Initialize variable j with a floating-point value of 1
j = 1.0

# Calculate initial value for k using the formula (j + 13) / 27
k = (j + 13) / 27.0

# Loop while k is less than or equal to 10
while k <= 10
    # Increment k by 1.2 in each iteration
    k += 1.2
    
    # Calculate the value of i using the formula: i = 3 * k - 1
    i = 3 * k - 1
    
    # Print the current values of k and i
    puts "K: #{k}, i: #{i}"
end
